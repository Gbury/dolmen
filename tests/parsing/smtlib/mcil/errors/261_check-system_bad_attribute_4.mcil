(check-system sys :reachable (r (= 1 1)) :trans)