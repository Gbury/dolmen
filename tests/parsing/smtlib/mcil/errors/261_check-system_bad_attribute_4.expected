File "tests/parsing/smtlib/mcil/errors/261_check-system_bad_attribute_4.mcil", line 1, character 41-47:
1 | (check-system sys :reachable (r (= 1 1)) :trans)
                                             ^^^^^^
Error while parsing a check-system command, read the reserved word ':trans',
      but expected an attribute from the set (:assumption, :reachable,
      :query, :queries) or a closing parenthesis.
